### PR TITLE
Pre-trained model

### PR DESCRIPTION
**Pull request by gngdb**
Trained on ImageNet with the default settings. Looks like I should've read the paper more carefully first though, there are some details to the training I messed up: weight decay was set wrong, batch size was too small, learning rate schedule was wrong.

Details:

* Trained for 90 epochs, best results were at epoch 83.
* Batch size was 256.
* Learning rate 0.1, schedule was multiply by 0.1 at epoch 30 and epoch 60.
* Momentum 0.9.
* Weight decay `1e-4`.
* Took about a week on a single Titan X.

I'll probably start it training again with (closer to) the correct settings.

### CODE REVIEW

**Comment by jaxony** [comment]
@gngdb Thanks for the PR! I'll test it out tomorrow and merge it in if it's all good.
